### PR TITLE
[LUN-3393] Add a vessel-tracking how-to guide

### PR DESCRIPTION
**Pull request by jstasiak**
This is a recently introduced feature and we figured we could use a dedicated how-to guide for it to lower the barrier to entry.

Resolves: https://linear.app/lune/issue/LUN-3393/write-guide-for-docs

### CODE REVIEW

**Comment by jstasiak** [comment]
@rbruggem So this is somewhat experimental – I went with a document structure differing from the existing guides:

* Internal ToC enabled
* No React components, no HTML, no MDX extensions – pure Markdown

Comments and feedback much appreciated.

**Comment by jstasiak** [comment]
Preview: https://new-how-to.lune-docs-azp.pages.dev/guides/how-to-vessel-track

@rbruggem do you have any ideas as to why the sidebar on this page is missing?

**Review by charcarr — APPROVED** [review]
Love it Jakub. So clear and concise.

**Comment by jstasiak** [comment]
A few things to adjust a little bit before merging:

* The missing sidebar
* I'm not quite happy with requiring `jq` here, alas our API responses aren't pretty-printed by default (maybe they should be?)
* Is it obvious that the snippets at the end are part of a single response JSON?

File: docs/guides/how-to-vessel-track.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    },
+    "shipment": {
+        "containers": "2"
+    }
+}' | jq
+```
+
+## Interpreting the response
+
+If everything goes well you should see something like below (showing only the relevant parts
```
[the remark this answers is not in the document]
Comment (jstasiak) [reply docs/guides/how-to-vessel-track.md:97]: Not needed per se, just feel slightly better to me

File: docs/guides/how-to-vessel-track.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+used to calculate the distance for better results compared to other, less precise distance
+calculation methods.
+
+## Prerequisites
+
+In order to proceed with the steps provided you need
+
+* A Lune API key in the `LUNE_API_KEY` environment variable
+* Vessel tracking enabled in your Lune organisation
+* cURL and jq installed (or be willing to adapt the procedure to your tools)
```
[the remark this answers is not in the document]
Comment (jstasiak) [reply docs/guides/how-to-vessel-track.md:13]: Yeah good question, not needed strictly speaking

**Comment by jstasiak** [comment]
Things to resolve before merging (after a chat with @rbruggem):

* Fix the sidebar (obviously)
* Remove the table of contents on the right side (I like it but it's inconsistent with the rest of the documentation)
* Use the request/response snippet components if possible

Explicitly not bringing two-column layout to this page, we're willing to explore a slightly different format with this new guide.

**Comment by jstasiak** [comment]
@rbruggem https://github.com/lune-climate/lune-docs/pull/919/commits/0e1b2953ae3319a5333025fe638069add5fbcef9 🤷‍♀️

**Pull request merged**